Methods Section - Fill in missing links section (#12)
greenelab/annorxiver_manuscript: content/03.methods.md

We used principal component analysis (PCA) [@doi:10.1111/1467-9868.00196] to project bioRxiv document vectors into a low dimensional space.
We trained this model using scikit-learn's [@raw:scikit-learn] implementation of a randomized solver [@arxiv:0909.4061] with a random seed of 100, output of 50 principal components, and default settings for all other parameters.
For each principal component we calculated its cosine similarity with  all tokens in our word2vec model's vocabulary.
We report the top 100 positive and negative scoring tokens in the form of  word clouds, where the size of each word corresponds to the magnitude of similarity and color represents positive (blue) or negative (orange) association.

> ### Mending Missing Preprints to Published Versions Links

Reviewer: ```suggestion
### Discovering Unannotated Preprint-Publication Relationships
```

> Linking preprints to their published versions is not a flawless process as preprints can drastically change as they undergo the revision process.

Reviewer: ```suggestion
Automated procedures are in place to link preprints to peer reviewed versions and many journals require authors to update preprints with a link to the published version.
However, automated procedures at _bioRxiv_ are often based on exact matching of certain attributes and authors can forget to establish a link after publication.
```

> For example, preprints can change their titles right before publication (e.g. [@doi:10.1101/376665] and [@doi:10.1242/bio.038232]), which results in both the preprint and its published version being treated as distinct despite being the same article. 

Reviewer: ```suggestion
For example, authors can change the title between a preprint and published version (e.g., [@doi:10.1101/376665] and [@doi:10.1242/bio.038232]), which prevents _bioRxiv_ from automatically establishing a link.
If the authors do not report the publication to _bioRxiv_, the preprint and the published version are treated as distinct entities despite representing the same underlying research. 
```

> We sought to remedy this issue by patching in missing preprint to published version links.

Reviewer: ```suggestion
We recognized that the distance in embedding space could reveal preprint to published version links that were missed by existing automated processes.
```

> First, we used CrossRef [@doi:10.1629/uksg.233] to identify bioRxiv preprints  that already have links to their corresponding published version.

Reviewer: ```suggestion
First, we used CrossRef [@doi:10.1629/uksg.233] to identify bioRxiv preprints that were linked to a corresponding published article.
```

We filtered out links that contained papers not in PubMed Central's Open Access corpus.

> Following the preprocessing step, we calculated the distribution of known preprint to published distances by taking the Euclidean distance between preprint embeddings and their corresponding published version embeddings.

Reviewer: ```suggestion
Following the preprocessing step, we calculated the distribution of known preprint to published distances by taking the Euclidean distance between the preprint's embedding coordinates and the coordinates of the published version.
```

> We also calculated the null distribution which consisted of distances between preprints and a randomly selected article from the same venue as their corresponding published version.

Reviewer: ```suggestion
We also calculated a background distribution, which consisted of the distance between each preprint with an annotated publication and a randomly selected article from the same journal the published version.
```

> Next, we calculated distances between preprints without a published version link with PubMed Central articles that aren't matched with a corresponding preprint.

Reviewer: ```suggestion
Next, we calculated distances between preprints without a published version link with PubMed Central articles that weren't matched with a corresponding preprint.
```

> We filtered out distances that were greater than the min value of our null distribution as these are considered false positives.

Reviewer: ```suggestion
We filtered any potential links with distances that were greater than the minimum value of the background distribution to reduce the curation burden.
```

> Lastly, we binned the remaining pairs based on percentiles of our true distribution and randomly sampled 200 articles from each bin for manual annotation.

Reviewer: ```suggestion
Lastly, we binned the remaining pairs based on percentiles from the annotated pairs at the [0,25th percentile), [25th percentile, 50th percentile), [50th percentile, 75th percentile), and [75th percentile, minimum background distance).
We randomly sampled 50 articles from each bin for manual annotation.
We shuffled these four sets to produce a list of 200 potential preprint-published pairs with a randomized order.
We supplied these to two scientists to manually determine if each link between a preprint and a putative matched version was correct or incorrect.
After curation in duplicate there were eight mismatches.
These eight were supplied to a third scientist, who carefully reviewed each case and made a final determination.
These provided the set that we used to evaluate the extent to which distance in the embedding space revealed true but unannotated links between preprints and their published verisons.
```
